Tests/Shared.Specs/FormatterSpecs.cs

```diff
@@ -748,6 +748,65 @@ public void When_formatting_a_completion_source_it_should_include_the_underlying
             result.Should().Match("*TaskCompletionSource*Task*System.Int32*Status=WaitingForActivation*");
         }
 
+        [Theory]
+        [MemberData(nameof(MultiDimentionalArrayData))]
+        public void When_formatting_a_multi_dimentional_array_it_should_show_structure(object value, string expected)
```

Comment:
Can you please move these into a dedicated specs class? Most of what's here is pretty old and still needs to be split.